ppdet/data/source/voc.py

```diff
@@ -110,7 +110,11 @@ def load_roidb_and_cname2cid(self):
                     break
                 img_file, xml_file = [os.path.join(image_dir, x) \
                         for x in line.strip().split()[:2]]
+                if not os.path.exists(img_file):
+                    logger.warn('Illegal image file: {}'.format(img_file))
```

Comment:
`Illegal image file: {}, and will be ignored.`

Reply:
Done